doc/index.rst

```diff
@@ -1,69 +1,94 @@
+#########################
+Electrometer
+#########################
+
+.. image:: https://img.shields.io/badge/SAL-API-gray.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/Electrometer.html
+.. image:: https://img.shields.io/badge/GitHub-gray.svg
+    :target: https://github.com/lsst-ts/ts_electrometer2
+.. image:: https://img.shields.io/badge/Jira-gray.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_electrometer2
+.. image:: https://img.shields.io/badge/Jenkins-gray.svg
+    :target: https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/ts_electrometer2/
+
+.. TODO: Delete the note when the page becomes populated
+
+.. Warning::
```

Comment:
I think you can remove this now... or at least remove "and not ready for active use"